docs/faq.md

```diff
@@ -61,3 +61,11 @@ python3 -m cibuildwheel --output-dir wheelhouse
 python3 -m pip install twine
 python3 -m twine upload wheelhouse/*.whl
 ```
+
+### 'ImportError: DLL load failed: The specific module could not be found' error on Windows
+
+Visual Studio and MSVC link the compiled binary wheels to the Microsoft Visual C++ Runtime. Normally, these are included with Python, but when compiling with a newer version of Visual Studio, it is possible users run into problems on system that do not have these runtime libraries installed. The solution is to download the corresponding Visual C++ Redistributable from the [Microsoft website](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
```

Comment:
```suggestion
Visual Studio and MSVC link the compiled binary wheels to the Microsoft Visual C++ Runtime. Normally, these are included with Python, but when compiling with a newer version of Visual Studio, it is possible users will run into problems on systems that do not have these runtime libraries installed. The solution is to download the corresponding Visual C++ Redistributable from the [Microsoft website](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
```

Reply:
This is only a problem with 2.7, right? Python 3.5 was compiled with 14.x, so 3.5-3.9 do not have an issue; Python 3.5+ always includes `VCRUNTIME140.dll`? (see table https://wiki.python.org/moin/WindowsCompilers for reference)

Edit: yes, this is said below; I would reword this a little to clarify that this is only a problem for Python 2.7 on Windows when compiled with a non 2008 compiler. Otherwise, I think the MSVC redistributable must be present to run Python itself. (and of course, the problem below)

Reply:
No. Problem is that VisualStudio 2019 Some times links against `VCRUNTIME140_1.dll` next to `VCRUNTIME140.dll` Which is only in modern version of Microsoft Visual C++ Redistributable. older ons contains only  `VCRUNTIME140.dll`

Reply:
That's what I meant by "the problem below". This seems to indicate that `VCRUNTIME140.dll` will sometimes be missing; whereas it will only possibly be missing in the single case where we use the Python 2.7 hack (like in pybind11). Python 3.5+ require `VCRUNTIME140.dll` to run, so it can't be missing.

The paragraph below is fine.

Reply:
@henryiii Thanks, yes. I meant to give a generic overview of the situation first. Somehow, this is not always clear when looking for issues. I'll make a link to the Python 2.7 section, though (we already have this, but it's in the "modern C++" part of the docs, because that's where a modern VS is needed).

Reply:
Added an extra sentence and some additional links. I do feel like I'm repeating myself a bit, now? Any suggestions on how to collapse sentences, or is this fine?